recipes/openssh/openssh.inc

```diff
@@ -70,15 +70,18 @@ do_install_extra_files() {
 
 inherit auto-package-utils
 AUTO_PACKAGE_UTILS = "ssh-add ssh-agent ssh-keyscan ssh-keygen ssh-keysign \
-	scp ssh sshd sftp sftp-server ssh-pkcs11-helper"
+	scp ssh sshd sftp sftp-server ssh-pkcs11-helper ssh-config sshd-config"
 
 ALLOW_EMPTY_${PN} = "1"
-FILES_${PN}-ssh = "${sysconfdir}/ssh/ssh_config ${bindir}/slogin"
+FILES_${PN}-ssh = "${bindir}/slogin"
 FILES_${PN}-ssh-doc = "${mandir}/*/ssh_config.* ${mandir}/*/slogin.*"
 FILES_${PN}-sshd = "${sysconfdir}/init.d/sshd \
-	${sysconfdir}/ssh/moduli ${sysconfdir}/ssh/sshd_config \
+	${sysconfdir}/rcS.d/* \
+	${sysconfdir}/ssh/moduli \
 	${localstatedir}/run/sshd"
 FILES_${PN}-sshd-doc = "${mandir}/*/moduli.* ${mandir}/*/sshd_config.*"
+FILES_${PN}-ssh-config = "${sysconfdir}/ssh/ssh_config"
+FILES_${PN}-sshd-config = "${sysconfdir}/ssh/sshd_config"
```

Comment:
you should make the ssh package depend on these two new packages to make sure we don't break stuff downstream. Probably best to let the new packages PROVIDE sshd_config and ssh_config respectively, and then depend on that, so it becomes easy to override those with your own little recipe.